app.py

```diff
@@ -28,12 +38,43 @@ def sign_up_page():
 
 @app.route('/sign_up', methods=['POST'])
 def process_sign_up():
-    if sign_up():
-        return "process completed Eden", 201
-    return "failed to create a new user", 401
+    if process_sign_up():
+        return render_template('signup_completed.html'), 200
+    return render_template('signup_railed.html'), 400
```

Comment:
`signup_failed.html`, not `signup_railed`.
Also: why is that in a new file? it's the same, but just with an error message, right?

Reply:
updated 'signup_railed' to 'signup_failed'.
we use an html file so we can use a button to go back to our index page, instead of manually changing the URL.